kitsune/wiki/templates/wiki/mobile/document.html

```diff
@@ -31,8 +31,13 @@
 {% endif %}
 
 {% block content %}
-  <div class="title-bar">{{ document_title(document) }}</div>
+  {% if breadcrumb_items[1] %}
+    <div class="filter-bar">
+      <a class="return-link" href="{{ breadcrumb_items[1][0] }}">{{ breadcrumb_items[1][1] }}</a>
```

Comment:
Ok. I'd expect this to want to go to the lowest level topic. That is, the last bread crumb (second to last?). The "dominant topic" language in the bug is to disambiguate the case where one article has multiple top level topics.